contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -6,11 +6,11 @@ import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
 import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
 
-import {GnosisAuction} from "../libraries/GnosisAuction.sol";
-import {OptionsThetaVaultStorage} from "../storage/OptionsVaultStorage.sol";
-import {Vault} from "../libraries/Vault.sol";
-import {VaultLifecycle} from "../libraries/VaultLifecycle.sol";
-import {ShareMath} from "../libraries/ShareMath.sol";
+import {GnosisAuction} from "../../libraries/GnosisAuction.sol";
+import {OptionsThetaVaultStorage} from "../../storage/OptionsVaultStorage.sol";
+import {Vault} from "../../libraries/Vault.sol";
+import {VaultLifecycle} from "../../libraries/VaultLifecycle.sol";
+import {ShareMath} from "../../libraries/ShareMath.sol";
 import {RibbonVault} from "./base/RibbonVault.sol";
 
 contract RibbonThetaVault is RibbonVault, OptionsThetaVaultStorage {
```

Comment:
Same here https://github.com/ribbon-finance/ribbon-v2/pull/47/files#r676741657

Reply:
we cannot do this because theta vaults inherit OptionsThetaVaultStorageV1 and delta vaults inherit OptionsDeltaVaultStorageV1.

both with different storage variables.

ribbon vault inherits OptionsVaultStorage which is the commonality between the two vaults